frontend/pages/about/contact-us.js

```diff
@@ -0,0 +1,267 @@
+import React, { useEffect, useState } from "react";
+import PageHeader from "../../components/header";
+import ContactForm from "../../components/contactForm";
+import styles from "../../styles/contactUs.module.css";
+import LoadingScreen from "../../components/LoadingScreen";
+import Image from "next/image";
+import Link from "next/link";
+
+const ContactUs = () => {
+  const [sourceLoading, setSourceLoading] = useState(true);
+  const [headerLoading, setHeaderLoading] = useState(true);
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, []);
+
+  // control when to stop loading
+  useEffect(() => {
+    setTimeout(() => {
+      setSourceLoading(false);
+    }, 1000);
+  }, []);
+
+  return (
+    <div>
+      {sourceLoading && headerLoading ? (
+        <LoadingScreen />
+      ) : (
+        <div>
+          {/* Cover Photo */}
+          <PageHeader
+            imgUrl="/headers/2021-exec-header.jpg"
+            title="Contact Us"
+            imageLoading={setHeaderLoading}
+          />
+          <div className={styles.form}>
+            <h2 className={styles.inquiry}>Send Through a General Inquiry</h2>
+            <div className={styles.formRight}>
+              <Image
+                src={
+                  window.matchMedia &&
+                  window.matchMedia("(prefers-color-scheme: dark)").matches
+                    ? `${"/willow_contact_dark_mode.png"}`
+                    : `${"/willow_contact.png"}`
```

Comment:
Think there is a bug here where the images don't change based on mode i.e. willow's phone should black when it is in light mode and orange in dark mode